ui/app/templates/release_scheduled_change_modal.html

```diff
@@ -6,14 +6,21 @@ <h3 class="modal-title" ng-show="sc.name && sc.when">Edit Scheduled Change for {
 <div class="modal-body">
   <form role="form">
     <h4>Scheduled Change Details</h4>
-
       <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+            <div class="help-block">
+                <div ng-hide="hideAsapInfo" class="alert alert-info" auto-hide="3000">
+                    <a href="#" class="close" arial-label="close" ng-click="hideAsapInfo=true">&times;</a>
+                    <strong>Info !</strong> Use ASAP to make changes as soon as possible
+            </div>
+        </div>
```

Comment:
I don't think this notification in particularly useful. "ASAP" is a pretty well understood term already, so an additional explanation isn't really helpful.